sonic_installer/main.py

```diff
@@ -184,7 +184,7 @@ def install(url):
 
     if get_image_type() == IMAGE_TYPE_ABOOT:
         run_command("/usr/bin/unzip -od /tmp %s boot0" % image_path)
-        run_command("swipath=%s target_path=/host sonic_upgrade=1 . /tmp/boot0" % image_path)
+        run_command("swipath=%s target_path=/host install=y . /tmp/boot0" % image_path)
```

Comment:
are you going to change boot0.j2 in sonic-buildimage to support install option? current I only see sonic_upgrade there.

Reply:
you do not really need to change to install=y. keep it as it is is better, so that we can merge your PR without the boot0.j2 change. there is no coordination, otherwise, we need to coordinate this PR with the boot0.j2 PR.

Reply:
I was planning on the pull request on sonic-buildimage that bumps the sonic-utilities and adds the boot0.j2 change as soon as this one merged.
But you are correct I can leave the sonic_upgrade=1 as it is. Will update this PR.